src/Schedulers/CustomersScheduler.php

```diff
@@ -120,6 +134,19 @@ public static function schedule_import( $user_id ) {
 		self::schedule_action( 'import', array( $user_id ) );
 	}
 
+	/**
+	 * Schedule an action to anonymize a single Order.
+	 *
+	 * @param WC_Order $order Order object.
+	 * @return void
+	 */
+	public static function schedule_anonymize( $order ) {
+		if ( is_a( $order, 'WC_Order' ) ) {
+			// Postpone until any pending updates are completed.
+			self::schedule_action( 'anonymize', array( $order->get_id() ), time() + MINUTE_IN_SECONDS + 5 );
```

Comment:
The pending update is the customer update hook which pulls the user info from the WP account. WC anonymizes the order data but not the user account so some of the data is re-populated. To prevent it from being repopulated we should remove the WP user ID as well.

Reply:
Is this a scheduled action hook?  Could we add it under `get_dependencies()` or is there a reason it doesn't work here?

Reply:
> Could we add it under get_dependencies()

Done in `6d5a06d`.